requirements/udata.pip

```diff
@@ -39,7 +39,7 @@ bleach==3.3.1
 blinker==1.4
     # via
     #   flask-mail
-    #   flask-navigation
+    #   flask-navigation-temp-python-311
```

Comment:
Why wasn't the modification done sooner? :thinking:

Reply:
Don't know… I reverted the change…